cradle-core/src/main/java/com/exactpro/cradle/BookInfo.java

```diff
@@ -403,7 +394,12 @@ private IPageInterval getPageInterval(Instant timestamp) {
         Instant start = toInstant(epochDay);
         Instant end = start.plus(1, ChronoUnit.DAYS).minus(1, ChronoUnit.NANOS);
         Collection<PageInfo> loaded = pagesLoader.load(id, start, end);
-        return loaded.isEmpty() ? null : create(id, start, cacheName, loaded);
+        if (loaded.isEmpty()) {
+            return null;
+        }
+        // We shouldn't register metric to avoid the `loads` >> `requests` case
```

Comment:
Could you please advise what this means exactly? I am looking at the code and I have no clue how to interpret this sentence

Reply:
I added this comment when work on the dashboard for hit / miss rates. But I think I doesn't matter.

Reply:
The point is that it is hard to understand what this comment tries to say) Even after your comment here I still cannot get it... I think if we leave something in the code it should be clear what it says. More than one line can be used to express what you mean.
This concerns me because if the task is assigned to somebody else he or she needs all the information to continue the work. And our responsibility to leave this information either in the code or in the issue. And this information should not require decryption)

Reply:
I understood you point and try to solve short comment in future.

Reply:
The origin comment should be 
We shouldn't register metric for empty page intervals to avoid the `loads` >> `requests` case
When loads number more than requests number, current dashboard for calculating hit / miss rate shows wrong values